client/asset/eth/polygon_bridge.go

```diff
@@ -0,0 +1,452 @@
+package eth
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	"decred.org/dcrdex/dex"
+	"decred.org/dcrdex/dex/dexnet"
+	"decred.org/dcrdex/dex/networks/erc20"
+	"decred.org/dcrdex/dex/networks/erc20/polygonbridge"
+)
+
+var (
+	usdtEthID, _     = dex.BipSymbolID("usdt.eth")
+	usdtPolygonID, _ = dex.BipSymbolID("usdt.polygon")
+	wethPolygonID, _ = dex.BipSymbolID("weth.polygon")
+	ethID, _         = dex.BipSymbolID("eth")
+	polygonID, _     = dex.BipSymbolID("polygon")
+	ethMaticID, _    = dex.BipSymbolID("matic.eth")
+)
```

Comment:
Three of these are already global variables in eth.go starting at line 122:
```
	usdcTokenID, _  = dex.BipSymbolID("usdc.eth")
	usdtTokenID, _  = dex.BipSymbolID("usdt.eth")
	maticTokenID, _ = dex.BipSymbolID("matic.eth")
```

Can just use these new declarations there.